Workarounds/BreathOfTheWild_LwzxNullCheck_VK/rules.txt

```diff
@@ -0,0 +1,5 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+path = "The Legend of Zelda: Breath of the Wild/Workarounds Vulkan/LWZX Crash"
```

Comment:
Is this pack really necessary for Vulkan? I've been playing the game in Vulkan without this pack and have encountered zero crashes.

If you want to keep it though, fix the metadata.
```
name = "LWZX Crash"
path = "The Legend of Zelda: Breath of the Wild/Workarounds/LWZX Crash for Vulkan"
```

Also, don't include this crash workaround in all the packs.

Reply:
it was needed for if users dont have Full sync at GX2 drawdone on the game can crash. we cant really test it in vulkan as we need it on for the games to run, but in opengl its needed any time they have it off.